pkg/providers/instance/instance.go

```diff
@@ -152,9 +152,10 @@ func (p *Provider) List(ctx context.Context) ([]*Instance, error) {
 }
 
 func (p *Provider) Delete(ctx context.Context, id string) error {
-	if _, err := p.ec2Batcher.TerminateInstances(ctx, &ec2.TerminateInstancesInput{
+	response, err := p.ec2Batcher.TerminateInstances(ctx, &ec2.TerminateInstancesInput{
```

Comment:
```suggestion
	res, err := p.ec2Batcher.TerminateInstances(ctx, &ec2.TerminateInstancesInput{
```

nit: This is the name use elsewhere. I think we may also use `out` but you can check

Reply:
Haven't really used this elsewhere in this function but `out` seems good.